packages/blade-mcp/base-blade-template/src/ErrorFallback.tsx

```diff
@@ -0,0 +1,57 @@
+import { Alert, Box, Heading, Text } from '@razorpay/blade/components';
+import { useEffect, useState } from 'react';
+const ErrorFallback = ({ error }: { error: Error; resetErrorBoundary?: () => void }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  useEffect(() => {
+    if (isCopied) {
+      setTimeout(() => {
+        setIsCopied(false);
+      }, 2000);
+    }
+  }, [isCopied]);
+
+  return (
+    <Box
+      textAlign="center"
+      display="flex"
+      flexDirection="column"
+      alignItems="center"
+      justifyContent="center"
+      height="100vh"
+      width="100vw"
+    >
+      <Box
+        display="inline-flex"
+        gap="spacing.4"
+        flexDirection="column"
+        maxWidth="80%"
+        width="500px"
+      >
+        <Heading as="h1" size="xlarge">
+          There is a runtime error 🫣
+        </Heading>
+        <Text size="large">But don't worry, AI can fix it 💪🏼</Text>
+        <Alert
+          color="negative"
+          isFullWidth
+          description={`Runtime Error: ${error.message}`}
+          isDismissible={false}
+          actions={{
+            primary: {
+              text: isCopied ? 'Copied' : 'Copy Error',
+              onClick: () => {
+                const prompt = `Fix the following runtime error: ${error.message}\n\nstack: ${error.stack}`;
+                navigator.clipboard.writeText(prompt);
+                setIsCopied(true);
+              },
+            },
+          }}
+        />
+        <Text>You can ask AI to "Fix runtime errors" or copy paste this error in chat</Text>
```

Comment:
do we need this since our copy error already has it? maybe copy CTA should be enough to make it easy? we can just mention copy the error in the chat?

Reply:
It doesn't lead to much maintainence anyways so more of "good to have". Also simpler to fix errors this way as you don't have to pass any context of error message and all.